assets/pages/contributions-landing/newContributionsLanding.jsx

```diff
@@ -74,12 +75,50 @@ const countryGroupSpecificDetails: {
+              <span className="countries__label">{selectedCountryGroup.name} ({currencies[selectedCountryGroup.currency].extendedGlyph})</span>
+              <span className="icon icon--arrows">
+                <svg width="19" height="10" xmlns="http://www.w3.org/2000/svg"><path className="icon__arrow-down" d="M1.25.5l-.75.75L9 9.5h.95l8.55-8.25-.75-.75L9.5 7.25z" /><path className="icon__arrow-up" d="M1.25 9.5L.5 8.75 9 .5h.95l8.55 8.25-.75.75L9.5 2.75z" /></svg>
+              </span>
+            </summary>
+            <ul className="countries__list">
+              {(Object.values(countryGroups): any).map((g: CountryGroup) => (
+                <li className="countries__item">
+                  <a href={`/${g.supportInternationalisationId}/contribute.react`}>
+                    {g === selectedCountryGroup ? (
+                      <span className="icon">
+                        <svg width="18" height="14" xmlns="http://www.w3.org/2000/svg"><path d="M1.36 6.9l-.86.888L4.798 14h.409L17.5.865 16.64 0 5.207 10.694z" /></svg>
+                      </span>
+                    ) : ''}
+                    {g.name} ({currencies[g.currency].extendedGlyph})
+                  </a>
+                </li>
+              ))}
+            </ul>
+          </details>
+        </header>
+      }
+      footer={
+        <footer role="contentinfo" className="gu-content__footer">
+          <div className="license">
+            <small className="license__txt">© 2018 Guardian News and Media Limited or its affiliated companies. All rights reserved.</small>
+            <p>The ultimate owner of the Guardian is The Scott Trust Limited, whose role it is to secure the editorial and financial independence of the Guardian in perpetuity. Reader contributions support the Guardian’s journalism. Please note that your support of the Guardian’s journalism does not constitute a charitable donation, as such your contribution is not eligible for Gift Aid in the UK nor a tax-deduction elsewhere. If you have any questions about contributing to the Guardian, please <a href="mailto:contribution.support@theguardian.com">contact us here</a>.</p>
+          </div>
+        </footer>
```

Comment:
You're one step away from being able to use the existing `Footer` component, I think this should give you the same content (and will internationalise the contact email for you):

```jsx
<Footer disclaimer countryGroupId={countryGroupId} />
```

**Note:** The footer component doesn't currently include `role="contentinfo"`, it probably should, this is a nice improvement.